README.md

```diff
@@ -21,7 +21,7 @@ Once awakened, the bot will download the diff of the pull request and figure out
 
 <img width="1139" src="https://cloud.githubusercontent.com/assets/197597/11022818/92edd20e-861d-11e5-8c44-c64c1a7de79f.png">
 
-For these, it will download the associated blame to figure out who last touched that line last as he may be a good reviewer.
+For these, it will download the associated blame to figure out who last touched that line last as they may be a good reviewer.
```

Comment:
"he/she" still kind of assumes only two genders. Non-gendered forms are more inclusive.